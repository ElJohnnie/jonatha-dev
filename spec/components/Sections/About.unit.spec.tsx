import AboutView from '@/app/[locale]/about/view/about.view';
import { render } from '@testing-library/react';
import { AcademicCapIcon } from '@heroicons/react/20/solid';

const aboutMockup = {
  title: 'title',
  description: 'description',
  postDescription: 'post-description',
  features: [
    {
      name: 'Universidade de Santa Cruz do sul:',
      description: 'Teste',
      icon: AcademicCapIcon,
    },
  ],
  image: true,
};

describe('should render correctly About component', () => {
  it('match snapshot', () => {
    const { container } = render(<AboutView {...aboutMockup}></AboutView>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
