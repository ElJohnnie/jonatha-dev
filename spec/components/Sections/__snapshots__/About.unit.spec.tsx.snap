// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly About component match snapshot 1`] = `
<section
  class="mt-4 h-auto pt-4"
>
  <div
    class="mx-auto grid max-w-screen-xl px-4 py-8 lg:grid-cols-12 lg:py-16"
  >
    <div
      class="mr-auto md:col-span-7"
    >
      <div
        class="top-[12vh] hidden h-auto place-self-start md:fixed lg:mt-0 lg:flex"
      >
        <img
          alt="profile"
          class="filter-grayscale"
          data-nimg="1"
          decoding="async"
          height="550"
          loading="lazy"
          src="/_next/image?url=%2Fimages%2Fjonatha.png&w=1200&q=75"
          srcset="/_next/image?url=%2Fimages%2Fjonatha.png&w=640&q=75 1x, /_next/image?url=%2Fimages%2Fjonatha.png&w=1200&q=75 2x"
          style="color: transparent;"
          width="550"
        />
      </div>
    </div>
    <div
      class="mr-auto place-self-center md:col-span-5 lg:max-w-lg"
    >
      <h2
        class="text-base font-semibold leading-7 text-indigo-600"
      >
        title
      </h2>
      <p
        class="mt-2 text-3xl font-bold tracking-tight dark:text-white sm:text-4xl"
      >
        description
      </p>
      <p
        class="mt-6 text-lg leading-8 text-gray-300"
      >
        post-description
      </p>
      <dl
        class="mt-10 max-w-xl space-y-8 text-base leading-7 text-gray-300 lg:max-w-none"
      >
        <div
          class="relative pl-9"
        >
          <dt
            class="inline font-semibold text-white"
          >
            <svg
              aria-hidden="true"
              class="absolute left-1 top-1 h-5 w-5 text-indigo-600"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M9.664 1.319a.75.75 0 01.672 0 41.059 41.059 0 018.198 5.424.75.75 0 01-.254 1.285 31.372 31.372 0 00-7.86 3.83.75.75 0 01-.84 0 31.508 31.508 0 00-2.08-1.287V9.394c0-.244.116-.463.302-.592a35.504 35.504 0 013.305-2.033.75.75 0 00-.714-1.319 37 37 0 00-3.446 2.12A2.216 2.216 0 006 9.393v.38a31.293 31.293 0 00-4.28-1.746.75.75 0 01-.254-1.285 41.059 41.059 0 018.198-5.424zM6 11.459a29.848 29.848 0 00-2.455-1.158 41.029 41.029 0 00-.39 3.114.75.75 0 00.419.74c.528.256 1.046.53 1.554.82-.21.324-.455.63-.739.914a.75.75 0 101.06 1.06c.37-.369.69-.77.96-1.193a26.61 26.61 0 013.095 2.348.75.75 0 00.992 0 26.547 26.547 0 015.93-3.95.75.75 0 00.42-.739 41.053 41.053 0 00-.39-3.114 29.925 29.925 0 00-5.199 2.801 2.25 2.25 0 01-2.514 0c-.41-.275-.826-.541-1.25-.797a6.985 6.985 0 01-1.084 3.45 26.503 26.503 0 00-1.281-.78A5.487 5.487 0 006 12v-.54z"
                fill-rule="evenodd"
              />
            </svg>
            Universidade de Santa Cruz do sul:
          </dt>
           
          <dd
            class="inline"
          >
            Teste
          </dd>
        </div>
      </dl>
    </div>
  </div>
</section>
`;
